jerry-core/ecma/builtin-objects/ecma-builtin-bigint-prototype.c

```diff
@@ -86,7 +98,7 @@ ecma_builtin_bigint_prototype_object_to_string (ecma_value_t this_arg, /**< this
                                                 const ecma_value_t *arguments_list_p, /**< arguments list */
                                                 uint32_t arguments_list_len) /**< number of arguments */
 {
-  ecma_value_t bigint = ecma_builtin_bigint_prototype_object_value_of (this_arg);
+  ecma_value_t bigint = this_arg;
```

Comment:
We could remove the "bigint" variable as it is basically the "this_arg" with this. In addition as there is no conversion here, do we need the ` ecma_free_value (bigint);` on line 130?